templates/mopro-example-app/android/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
yes it can be removed
thank you for finding this
[e865227](https://github.com/oskarth/mopro/pull/103/commits/e865227a4da54ce7ef8019fae49eb01837bf0cc2)